pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -150,6 +164,7 @@ var _ = Describe("Linseed rendering tests", func() {
 
 			// The deployment should have the hash annotation set, as well as a volume and volume mount for the client secret.
 			Expect(d.Spec.Template.Annotations["hash.operator.tigera.io/elastic-client-secret"]).To(Equal("ae1a6776a81bf1fc0ee4aac936a90bd61a07aea7"))
+			Expect(d.Spec.Template.Annotations["hash.operator.tigera.io/elastic-client-credential-secret"]).To(Equal("465c25d580ea36d8e7cda470a0c34afd05eae6f7:wq"))
```

Comment:
I didn't know you use vim :smile: . (I see a `:wq` which is causing CI to fail)

Reply:
u